.browserslistrc

```diff
@@ -1,11 +1,11 @@
-# Last 2 versions of each browsers, excluding dead browsers (e.g., BlackBerry)
-last 2 versions
-not dead
-
-# Browsers with more than 0.1% global usage, excluding old browsers
-> 0.1%
-not chrome < 45
-not ie < 9
-
-# Currently maintained Node.js versions
-maintained node versions
+# Last 2 versions of each browsers, excluding dead browsers (e.g., BlackBerry)
+last 2 versions
+not dead
+
+# Browsers with more than 0.1% global usage, excluding old browsers
+> 0.1%
+not chrome < 45
+not ie < 9
+
+# Currently maintained Node.js versions
+maintained node versions
```

Comment:
@kokopelli314 what changed in this file?

Reply:
Looks like it was just a formatting issue. Reverted the file.

Reply:
Sorry, this is probably on me 😢 I use Windows and have created file with CRLF line endings.